test/server/browser-provider-module-loader-test.js

```diff
@@ -0,0 +1,28 @@
+var BrowserProviderModuleLoader  = require('../../lib/browser/provider/module-loader');
```

Comment:
Maybe let's move new tests to the [browser-provider-test.js](https://github.com/DevExpress/testcafe/blob/master/test/server/browser-provider-test.js)?